### DISCUSSION TITLE
可以修改并分享您的主题吗？

### DISCUSSION THREAD

**Discussion by Sophomoresty** [Keldos-Li/typora-latex-theme#149] [Q&A] [answered]
我是一名四川大学的学生，非常喜欢您的主题。为匹配学校的论文格式，我修改了您的主题样式，不知道这种情况可不可以开源分享？

**Comment by Keldos-Li** [comment, marked as answer]
当然可以，这就是开源的意义所在哇～其实不需要询问或告知的～

另外，根据开源协议，最好可以在某处标写出使用部分代码的原始来源🥰

**Comment by Sophomoresty** [comment]
好的，我会标注的，非常感谢！
